corehq/apps/users/templates/users/partials/bootstrap5/edit_role_modal.html

```diff
@@ -871,13 +871,14 @@ <h4 class="modal-title" data-bind="text: modalTitle"></h4>
       <label data-bind="text: text"></label>
       <!-- ko if: id === 'access_web_apps' && hasRestrictedApplicationAccess -->
       <span class="help-block">
-        {% url "cloudcare_app_settings" domain as
-        manage_web_apps_permissions_url %} {% blocktrans %} This permission is
-        already configured via
-        <a target="_blank" href="{{ manage_web_apps_permissions_url }}"
-          >Manage Web Apps Permissions</a
-        >. Please remove this configuration before updating access here. {%
-        endblocktrans %}
+        {% url "cloudcare_app_settings" domain as manage_web_apps_permissions_url %}
+        {% blocktrans %}
+          This permission is already configured via
+          <a target="_blank" href="{{ manage_web_apps_permissions_url }}">
+            Manage Web Apps Permissions
+          </a>
+          . Please remove this configuration before updating access here.
+        {% endblocktrans %}
```

Comment:
I haven't figured out why it does this and seemingly only with `blocktrans` tags. I wonder if there's a parser config we could modify (probably not, Prettier intentionally doesn't expose many options).